nodejs/src/wrapper.ts

```diff
@@ -46,11 +47,74 @@ const { NetInstrumentation } = require('@opentelemetry/instrumentation-net');
 const { PgInstrumentation } = require('@opentelemetry/instrumentation-pg');
 const { RedisInstrumentation } = require('@opentelemetry/instrumentation-redis');
 
+// AWS lambda instrumentation response hook for Server-Timing support
+function getEnvBoolean(key: string, defaultValue = true) {
+  const value = process.env[key];
+
+  if (value === undefined) {
+    return defaultValue;
+  }
+
+  if (['false', 'no', '0'].indexOf(value.trim().toLowerCase()) >= 0) {
+    return false;
+  }
+
+  return true;
+}
+
+function appendHeader(response: any, header: string, value: string) {
```

Comment:
I had some difficulty understanding what is the `res` object in AWS lambda? 

If using express.js, is it the response object? If not using express (is it possible?), what is the response passed here?

By running OTel's test code the response was simply a string

Reply:
It can be anything (really, anything). String, custom type - whatever. By AWS examples and as an AWS API gateway idiom it should be an object with statusCode and body. Specifying headers will also work - it basically conforms to APIGatewayProxyResponseEvent model.